tests/jsx_fragment/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,50 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`fragment.js 1`] = `
+<></>;
```

Comment:
Might be worth adding a "complex" nested fragment example (here's the one from Babylon):

```
<div>
  <>
    <>
      <span>Hello</span>
      <span>world</span>
    </>
    <>
      <span>Goodbye</span>
      <span>world</span>
    </>
  </>
</div>
```